lib/turbo/test_assertions.rb

```diff
@@ -79,5 +79,76 @@ def assert_no_turbo_stream(action:, target: nil, targets: nil)
       selector << %([targets="#{targets}"]) if targets
       assert_select selector, count: 0
     end
+
+    # Assert that the rendered fragment of HTML contains a `<turbo-frame>`
+    # element.
+    #
+    # ==== Options
+    #
+    # * <tt>:id</tt> [String] matches the element's <tt>[id]</tt> attribute
+    # * <tt>:loading</tt> [String] matches the element's <tt>[loading]</tt>
+    #   attribute
+    # * <tt>:src</tt> [String] matches the element's <tt>[src]</tt> attribute
+    # * <tt>:target</tt> [String] matches the element's <tt>[target]</tt>
+    #   attribute
+    # * <tt>:count</tt> [Integer] indicates how many turbo frames are expected.
+    #   Defaults to <tt>1</tt>.
+    #
+    #   Given the following HTML fragment:
+    #
+    #     <turbo-frame id="example" target="_top"></turbo-frame>
+    #
+    #   The following assertion would pass:
+    #
+    #     assert_turbo_frame id: "example", target: "_top"
+    #
+    # You can also pass a block make assertions about the contents of the
+    # element. Given the following HTML fragment:
+    #
+    #     <turbo-frame id="example">
+    #       <p>Hello!</p>
+    #     </turbo-frame>
+    #
+    #   The following assertion would pass:
+    #
+    #     assert_turbo_frame id: "example" do
+    #       assert_select "p", text: "Hello!"
+    #     end
+    #
+    def assert_turbo_frame(id:, loading: nil, src: nil, target: nil, count: 1, &block)
+      selector = %(turbo-frame[id="#{id}"])
+      selector << %([loading="#{loading}"]) if loading
+      selector << %([src="#{src}"]) if src
+      selector << %([target="#{target}"]) if target
+      assert_select selector, count: count, &block
+    end
```

Comment:
What do you think about omitting the `count:` keyword option, and instead forwarding all remaining options:

```suggestion
    def assert_turbo_frame(id:, loading: nil, src: nil, target: nil, **options, &block)
      selector = %(turbo-frame[id="#{id}"])
      selector << %([loading="#{loading}"]) if loading
      selector << %([src="#{src}"]) if src
      selector << %([target="#{target}"]) if target
      assert_select selector, **options, &block
    end
```

Reply:
Since frames required IDs which should be unique, I thought it might make sense to set count to 1. Unspecified I believe it passes if 1+?